backend/bamboo/settings.py

```diff
@@ -10,6 +10,10 @@
 
 class BaseConfig:
     SECRET_KEY = os.getenv("SECRET_KEY", "dev key")
+    BAMBOO_MEDIA_DIR = os.getenv("BAMBOO_MEDIA_DIR", str(basedir / "media"))
```

Comment:
Can you make it under `data/`, this will be the old directory users need to mount into the container for generated content.

Reply:
ok